source/documentation/install-guide/chap-Installing_oVirt.html.md

```diff
@@ -10,13 +10,21 @@ Before you can configure and use the oVirt Engine, you must install the <package
 
 **Installing the oVirt Engine Packages**
 
-1. To ensure all packages are up to date, run the following command on the machine where you are installing the oVirt Engine:
+1. Before you can start installing the oVirt, add the official repository. Choose one based on your version requirement:
+
+        
+        # http://resources.ovirt.org/pub/yum-repo/ovirt-release35.rpm
```

Comment:
Thanks, this makes perfect sense. If this is now OK, can we merge this pull request?
It has the potential to save people from struggling.